tests/schemas/invalid/test_invalid_relationship_list_doesnt_exist.json

```diff
@@ -0,0 +1,68 @@
+{
+  "mime_type": "application/json/ons/eq",
+  "schema_version": "0.0.1",
+  "data_version": "0.0.3",
+  "survey_id": "0",
+  "title": "Test Relationships",
+  "theme": "default",
+  "description": "A questionnaire to test capturing of relationships.",
+  "messages": {},
+  "metadata": [{
+      "name": "user_id",
+      "type": "string"
+    },
+    {
+      "name": "period_id",
+      "type": "string"
+    },
+    {
+      "name": "ru_name",
+      "type": "string"
+    }
+  ],
+  "sections": [{
+    "id": "section",
+    "groups": [{
+      "id": "group",
+      "title": "Relationships",
+      "blocks": [{
+          "type": "RelationshipCollector",
+          "id": "relationships",
+          "title": "This will iterate over the people list, capturing the one way relationships.",
+          "for_list": "not-a-list",
+          "question": {
+            "id": "relationship-question",
+            "type": "General",
+            "title": "Thinking of {first_person_name}, {second_person_name} is their <em>...</em>",
+            "answers": [{
+              "id": "relationship-answer",
+              "mandatory": true,
+              "type": "Relationship",
+              "playback": "{second_person_name} is {first_person_name} <em>…</em>",
```

Comment:
These would be the possessive form of first_person_name, though I appreciate the transform is not done here anyway. I suppose it could test for it? Or test for the right number of placeholders? Maybe that's a bit inflexible, enforcing something we wouldn't always do.

Reply:
I think that's beyond the scope of the validator